### PR TITLE
Use webhook secrets from files

### PR DESCRIPTION
**Pull request by jhrozek**
# Summary

Instead of relying on the config to set them.

Relates to: https://github.com/stacklok/infra/issues/954

## Change Type

***Mark the type of change your PR introduces:***

- [ ] Bug fix (resolves an issue without affecting existing features)
- [x] Feature (adds new functionality without breaking changes)
- [ ] Breaking change (may impact existing functionalities or require documentation updates)
- [ ] Documentation (updates or additions to documentation)
- [ ] Refactoring or test improvements (no bug fixes or new functionality)

# Testing

I just verified that the env names are correct by running:
```
MINDER_WEBHOOK_CONFIG_WEBHOOK_SECRET_FILE=.secrets/webhook_secret MINDER_WEBHOOK_CONFIG_PREVIOUS_WEBHOOK_SECRET_FILE=.secrets/previous_secrets make run-server
```

# Review Checklist:

- [x] Reviewed my own code for quality and clarity.
- [ ] Added comments to complex or tricky code sections.
- [ ] Updated any affected documentation.
- [ ] Included tests that validate the fix or feature.
- [ ] Checked that related changes are merged.

### CODE REVIEW

File: deployment/helm/templates/deployment.yaml [hunk 1]
```diff
@@ -81,6 +81,10 @@ spec:
             value: "/secrets/github-app/github_app_webhook_secret"
           - name: "MINDER_PROVIDER_GITHUB_APP_FALLBACK_TOKEN_FILE"
             value: "/secrets/github-app/github_app_fallback_token"
+          - name: "MINDER_WEBHOOK_CONFIG_WEBHOOK_SECRET_FILE"
+            value: "/secrets/app/repo_webhook_secret"
```
Comment (eleftherias) [inline deployment/helm/templates/deployment.yaml:85]:
```suggestion
            value: "/secrets/app/github_repo_webhook_secret"
```

File: deployment/helm/templates/deployment.yaml [hunk 2]
```diff
@@ -81,6 +81,10 @@ spec:
...
+          - name: "MINDER_WEBHOOK_CONFIG_PREVIOUS_WEBHOOK_SECRET_FILE"
+            value: "/secrets/app/repo_fallback_webhook_secret"
```
Comment (eleftherias) [inline deployment/helm/templates/deployment.yaml:87]:
```suggestion
            value: "/secrets/app/github_repo_fallback_webhook_secret"
```

**Review by eleftherias — COMMENTED** [review]
Based on the values I see in infra, I think these are the right values

File: deployment/helm/templates/deployment.yaml (hunk 1 above)
Comment (jhrozek) [reply deployment/helm/templates/deployment.yaml:85]: 🤦🏻 of course I copied the AWS secret names and not the k8s secret names.
Thank you!

**Review by eleftherias — DISMISSED** [review]

**Comment by JAORMX** [comment]
Are we good to merge this?

**Comment by jhrozek** [comment]
> Are we good to merge this?

In general yes, I only didn't merge because I'm intermittently available today and I didn't want to force someone else to pick up the breakage in case this doesn't work.

**Comment by jhrozek** [comment]
@eleftherias @JAORMX could you please approve once again? I just resolved a conflict, no other changes

**Pull request merged**